lib/browser/init.ts

```diff
@@ -142,6 +142,9 @@ if (BUILDFLAG(ENABLE_REMOTE_MODULE)) {
 // Load protocol module to ensure it is populated on app ready
 require('@electron/internal/browser/api/protocol');
 
+// Load web-contents module to ensure it is populated on app ready
+require('@electron/internal/browser/api/web-contents');
```

Comment:
I'm surprised this is newly necessary—what changed to make this required?

Reply:
the de-structured assignment accessed require('electron').webContents, but now with TypeScript, it's delayed.